packages/frontend/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es6",
+    "target": "es2018",
```

Comment:
After looking into it, it seems all modern browsers support this except for maybe internet explorer